traffic_monitor/datareq/datareq.go

```diff
@@ -286,24 +287,47 @@ func WrapAgeErr(errorCount threadsafe.Uint, f func() ([]byte, time.Time, error),
 	}
 }
 
+func accessLogTime(t time.Time) float64 {
+	return float64(t.Unix()) + (float64(t.Nanosecond()) / 1e9)
+}
+
+func accessLogStr(
+	timestamp time.Time, // prefix
+	remoteAddress string, // addr
+	reqMethod string, // method
+	reqPath string, // path
+	reqProto string, // proto
+	statusCode int, // statusCode
+	respSize int, // respSize
+	reqServeTimeMs int, // reqServeTimeMs
+	userAgent string, // uas
+) string {
+	return fmt.Sprintf("%.3f addr=%s method=%s path=%s proto=%s statusCode=%d respSize=%d reqServeTimeMs=%d uas=%s", accessLogTime(timestamp), remoteAddress, reqMethod, reqPath, reqProto, statusCode, respSize, reqServeTimeMs, userAgent)
```

Comment:
Since this line is really long, should we put each parameter on its own line?

Also, I think it makes sense to mimic ATS logging fields here since that is what we do for other ATC components (TR at least):
```
%.3f chi=%s cqhm=%s url=\"%s?%s\" cqhv=%s pssc=%d b=%d ttms=%d uas=\"<user agent>\"
```
for the `url` field, notice I've quoted it and made it `%s?%s` which should consist of `r.URL.Path` and `r.URL.RawQuery` so that we have the full URL. Also, the uas field should be quoted as well.